### PR TITLE
Flaky specs: Use travel_back when travel_to is used

### PR DESCRIPTION
**Pull request by thejonroberts**
### What github issue is this PR for, if any?
None. Fixes flaky specs.

### What changed, and _why_?
Added travel_back wherever travel_to is used.

I was seeing a lot of spec failures locally when making a change that were seemingly unrelated to my changes. This fixed them.  A lot of places where travel to was used without a travel back. (Probably an assumption that this was done automatically). Could possibly do `travel_back` in an after(:each) block in spec_helper.rb instead. Would mean doing for every test, not just time-traveling specs, and I errored on the side of being explicit. The benefit of the spec helper approach would be that it won't ever be a problem again...

### CODE REVIEW

File: spec/requests/health_spec.rb
Comment (elasticspoon) [inline spec/requests/health_spec.rb:1]: This spec already has `travel_back in after blocks. I don't *think* we need additional ones?

**Review by elasticspoon — APPROVED** [review]
1 small comment but overall LGTM

File: spec/requests/health_spec.rb
Comment (thejonroberts) [reply spec/requests/health_spec.rb:1]: good catch, I removed them.

**Comment by elasticspoon** [comment]
@thejonroberts could you fix the lint on this?

**Comment by thejonroberts** [comment]
updated @elasticspoon

**Pull request merged**